jetpack.php

```diff
@@ -27,6 +27,24 @@
 defined( 'JETPACK_PROTECT__API_HOST' )       or define( 'JETPACK_PROTECT__API_HOST', 'https://api.bruteprotect.com/' );
 defined( 'JETPACK__WPCOM_JSON_API_HOST' )    or define( 'JETPACK__WPCOM_JSON_API_HOST', 'public-api.wordpress.com' );
 
+add_filter( 'rest_url_prefix', 'jetpack_index_permalinks_rest_api_url', 999 );
+/**
+ * Fix the REST API URL for sites using index permalinks
+ *
+ * @todo Remove when 4.7 is minimum version
+ * @see  https://core.trac.wordpress.org/ticket/38182
+ * @see  https://github.com/Automattic/jetpack/issues/5216
+ * @author kraftbj
+ **/
+function jetpack_index_permalinks_rest_api_url( $prefix ){
+	global $wp_rewrite, $wp_version;
+	if ( version_compare( $wp_version, '4.7-alpha-38790', '>=' ) && $wp_rewrite->using_index_permalinks() ){
```

Comment:
If my reasoning is correct, then this should be `<` instead of `>=`. Here's my reasoning:
We only need this for versions of WordPress that have not yet fixed that problem, so the version should actually be less than `4.7-alpha-38790`. This expression returns true if it's later than `4.7-alpha-38790`. Please let me know if I'm wrong.